### DISCUSSION TITLE
ACL Tags for Tailscale

### DISCUSSION THREAD

**Discussion by incaqueen** [almeidapaulopt/tsdproxy#50] [Q&A]
Hi there, has anybody implemented this with ACL tags? Right now I have ACLs with tags, so if it doesn't have a tag, all connections are dropped.

**Comment by almeidapaulopt** [comment]
have you tried and all connections are dropped?

**Reply by incaqueen** [reply]
No I haven't tried yet, just wanted to check before I went ahead.

Actually my question was answered on the Tailscale YT channel. The Auth Key has an option for a tag, so anything authenticated by that key will also receive the key tag. All good! I plan on testing it out this weekend.
